### DISCUSSION TITLE
Can someone help me use Vimeo videos with the React LightGallery component?

### DISCUSSION THREAD

**Discussion by mmalone-tongal** [sachinchoolur/lightGallery#1272] [Q&A] [answered]
Please, anyone that contributed to this library can you help me or add some documentation to follow which shows in detail how to use the `@vimeo/player` module with the `<LightGallery /> `React component?  There are docs showing how to use the other plugins but there is zero documentation for how to use this wonderful gallery with Vimeo.  The react component has a plugins prop for the other things such as videos and thumbnails etc, but nothing for Vimeo that I can see.

If this gallery takes a prop for Vimeo, please show me how.  I have searched for hours trying to complete a task and I have lost hope.  If someone out there knows how to use the React component with Vimeo videos so that they can play and pause based on the sliding of the gallery I will be forever grateful.  Currently I get an error that says I need the script, but nothing shows you how to implement it.  I installed the module via npm.  

Here is the error:
`lightGallery:- Make sure you have included //github.com/vimeo/player.js`

Thank you in advance for any help you can give.

**Comment by sachinchoolur** [comment, marked as answer]
Hey @mmalone-tongal,

Vimeo videos should work the same way other videos work. Just pass the Vimeo video URL via the src key and include the Video module.

Also, you need to include the `<script src="https://player.vimeo.com/api/player.js"></script>` tag 

Here is the demo (2nd slide) -  https://stackblitz.com/edit/lightgallery-react-update-slides?file=index.tsx

Let me know if you have any questions

**Reply by mmalone-tongal** [reply]
Hello @sachinchoolur, 

Thank you for the quick reply!  So will that also work for my `<LightGallery /> `component?  How does this component know that this script is available as a dependency?   Does having this script load globally resolve the issue?  I tried that.  Perhaps I did it wrong?

For example this is my use case.  I have plugins for videos and thumbnails etc:
```
<LightGallery
    thumbnail={true}
    plugins={[lgVideo, lgThumbnail, lgZoom]}
    download={false}
    selector=".light-gallery-item"
    addClass={styles.galleryStyles}
    licenseKey={license}
    autoplayVideoOnSlide
>
//Stuff goes here
</LightGallery>
```

I'll try adding that script again, but last time it had no effect.  The Vimeo player warning shows up regardless.  Is there something that I need to set via a prop for this React component from lighgallery.js?  I'll give the example a try!
